CONTRIBUTING.md

```diff
@@ -34,7 +34,7 @@ Code contributions to Iroh are greatly appreciated. Here is the general workflow
 
    - **General code guidelines**
 
-     - When possible, please document relevant pieces of code. If documentation refers to another object (module, function, struct, etc) use ``[`path::to::ReferencedObject`]`` to link it. For more information check the [rustdoc docummentation][rustdoc]
+     - When possible, please document relevant pieces of code following the [rust documentation conventions][docconventtions]. For more information on how the rust documentation system works check the [rustdoc docummentation][rustdoc].
```

Comment:
typo?

Reply:
i assume you meant the `[docconventtions]` with double `tt`?  Fixed that one, if there's another one let me know :)

Reply:
yes, sorry I meant that one :)